xLights/EffectsPanel.cpp

```diff
@@ -174,6 +189,9 @@ wxString EffectsPanel::GetRandomEffectStringFromWindow(wxWindow *w, const wxStri
 			s += AttrName + wxString::Format(wxT("%i"), v);
 		} else if (ChildName.StartsWith("ID_CHOICE")) {
             wxChoice* ctrl=(wxChoice*)ChildWin;
+            if (ctrl->GetCount() <= 0) {
```

Comment:
not sure exactly why but sometimes i noticed this would crash when creating lots of random effects from a timing track selection and the count was 0 which caused a divide by zero error below.